kvmagent/kvmagent/plugins/volume_cache_plugin.py

```
@@ -1043,20 +1043,33 @@ def stop(self):
+        return rsp
+
+    def _fill_pool_rsp(self, rsp, pool):
+        # type: (PoolRsp, PoolProcessor) -> None
         assert pool.vg and pool.lv and pool.fs and pool.mount_point
         rsp.poolUuid = pool.pool_uuid
         rsp.mountPoint = pool.mount_point[MountPointInfoFields.TARGET]
         try:
-            pool_capacity = pool.get_capacity()
-            rsp.capacity = pool_capacity.total
+            self._fill_pool_capacity(rsp, pool.get_capacity())
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)